File: onyx-remote-driver/src/main/java/com/onyx/client/exception/ConnectionFailedException.java
package com.onyx.client.exception;

/**
 * Created by tosborn1 on 2/13/17.
 *
 * Client failed to connect
 */
public class ConnectionFailedException extends OnyxServerException {

    public ConnectionFailedException()
    {

    }

    public ConnectionFailedException(@SuppressWarnings("SameParameterValue") String message)
    {
        this.message = message;
    }

    @Override
    public String getMessage() {
        return message;
    }
}

File: onyx-remote-driver/src/main/java/com/onyx/client/exception/ConnectionFailedException.kt
package com.onyx.client.exception

/**
 * Created by tosborn1 on 2/13/17.
 *
 * Client failed to connect
 */
class ConnectionFailedException(override val message: String? = "") : OnyxServerException(message) {

    companion object {
        @JvmField val CONNECTION_TIMEOUT = "Connection Timeout Occurred"
    }

}
